src/node.cc

```diff
@@ -779,6 +779,8 @@ static ExitCode InitializeNodeWithArgsInternal(
   // is security relevant, for Node it's less important.
   V8::SetFlagsFromString("--no-freeze-flags-after-init");
 
+  V8::SetFlagsFromString("--max_semi_space_size=64");
```

Comment:
Is it still possible to override it from the command line? Initialization is so spaghetti-like these days I have a hard time telling if this comes before or after V8::SetFlagsFromCommandLine().

Reply:
I think yes `V8::SetFlagsFromCommandLine()` is being called at `ProcessGlobalArgsInternal` which is called after this line

Reply:
@joyeecheung or @addaleax I think can validate if am correct